index.html

```diff
@@ -905,7 +907,59 @@ <h3>Join files together</h3>
+        <dt>[0:v:0][0:a:0]</dt><dd>selects the first video stream and first audio stream from the first input.<br>
+          Each reference to a specific stream is enclosed in square brackets. In the first stream reference, <code>0:v:0</code>, the first zero refers to the first input file, <code>v</code> means video stream, and the second zero indicates that it is the <i>first</i> video stream in the file that should be selected. Likewise, <code>0:a:0</code> means the first audio stream in the first input file.<br>
+          As demonstrated above, ffmpeg uses zero-indexing: <code>0</code> means the first input/stream/etc, <code>1</code> means the second input/stream/etc, and <code>4</code> would mean the fifth input/stream/etc.</dd>
+        <dt>[1:v:0][1:a:0]</dt><dd>As described above, this means select the first video and audio streams from the second input file.</dd>
+        <dt>concat=</dt><dd>starts the <code>concat</code> filter</dd>
+        <dt>n=2</dt><dd>states that there are two input files</dd>
+        <dt>:</dt><dd>separator</dd>
+        <dt>v=1</dt><dd>sets the number of output video streams.<br>
+          Note that this must be equal to the number of video streams selected from each segment.</dd>
+        <dt>:</dt><dd>separator</dd>
+        <dt>a=1</dt><dd>sets the number of output audio streams.<br>
+          Note that this must be equal to the number of audio streams selected from each segment.</dd>
+        <dt>[videoOut]</dt><dd>name of the concatenated output video stream. This is a variable name which you define, so you could call it something different, like “vOut”, “outv”, or “banana”.</dd>
+        <dt>[audioOut]</dt><dd>name of the concatenated output audio stream. Again, this is a variable name which you define.</dd>
+        <dt>"</dt><dd>quotation mark to end filtergraph</dd>
+        <dt>-map "[videoOut]"</dt><dd>map the concatenated video stream into the output file by referencing the variable defined above</dd>
+        <dt>-map "[audioOut]"</dt><dd>map the concatenated audio stream into the output file by referencing the variable defined above</dd>
+        <dt><i>output_file</i></dt><dd>path, name and extension of the output file</dd>
+      </dl>
+      <p>
+        To specify the characteristics of the output stream(s), add flags after the <code>-map "[out]"</code> part of the command.
+      </p>
+      <p>
+        For example, to ensure that the video stream of the output file is visually lossless H.264 with a 4:2:0 chroma subsampling scheme, the command above could be amended to include the following:<br>
+        <code>-map "[videoOut]" -c:v libx264 -pix_fmt yuv420p -preset veryslow -crf 18</code>
+      </p>
+      <p>
+        Likewise, to encode the output audio stream as mp3, the command could include the following:<br>
+        <code>-map "[audioOut]" -c:a libmp3lame -dither_method modified_e_weighted -qscale:a 2</code><br>
+      </p>
```

Comment:
A `<br>` immediately before the closing tag `</p>` is useless.

Reply:
Opps, the break tag was from when I had another line after that. Fixed in 73cd80b. Cheers @retokromer !